docs/ops/detection/PriorBox_1.md

```diff
@@ -82,80 +116,36 @@
 
     * **Description**: *fixed_ratio* is an aspect ratio of a box. For example, *fixed_ratio* equal to 2.000000 means that the aspect ratio for the first box aspect ratio is 2.
     * **Range of values**: a list of positive floating-point numbers
-    * **Type**: `float[]`
+    * **Type**: float[]
     * **Default value**: None
     * **Required**: *no*
 
 * *fixed_size*
 
     * **Description**: *fixed_size* is an initial box size (in pixels). For example, *fixed_size* equal to 15 means that the initial box size is 15.
     * **Range of values**: a list of positive floating-point numbers
-    * **Type**: `float[]`
+    * **Type**: float[]
```

Comment:
Removal of backticks should be reverted and other attributes should be re-checked to introduce backticks.  @ggalieroc please confirm that nothing has changed recently and we should use backticks for types in attributes section.

Reply:
Correct, the type section of attributes is surrounded by backticks